2016-03-22  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/alpha/fts.c: New file.
	* sysdeps/unix/sysv/linux/alpha/fts64.c: New file.

Index: glibc-2.28/sysdeps/unix/sysv/linux/alpha/fts.c
===================================================================
--- /dev/null
+++ glibc-2.28/sysdeps/unix/sysv/linux/alpha/fts.c
@@ -0,0 +1 @@
+#include <io/fts.c>
Index: glibc-2.28/sysdeps/unix/sysv/linux/alpha/fts64.c
===================================================================
--- /dev/null
+++ glibc-2.28/sysdeps/unix/sysv/linux/alpha/fts64.c
@@ -0,0 +1 @@
+#include <io/fts64.c>
